src/Common/MoistThermodynamics/MoistThermodynamics.jl

```diff
@@ -98,7 +99,7 @@ The air pressure, linearized around a dry rest state, from the equation of state
 and, optionally,
  - `q` [`PhasePartition`](@ref). Without this argument the results are that of dry air.
 """
-linearized_air_pressure(ρ::FT, e_tot::FT, e_pot::FT, q::PhasePartition{FT}=PhasePartition{FT}(FT(0), FT(0), FT(0))) where {FT<:Real} =
+linearized_air_pressure(ρ::FT, e_tot::FT, e_pot::FT, q::PhasePartition{FT}=q_pt_0(FT)) where {FT<:Real} =
   ρ*FT(R_d)*FT(T_0) + FT(R_d)/FT(cv_d)*(ρ*e_tot - ρ*e_pot - (ρ*q.tot - ρ*q.liq)*FT(e_int_v0) + ρ*q.ice*(FT(e_int_i0) + FT(e_int_v0)))
 
 linearized_air_pressure(e_kin::FT, e_pot::FT, ts::ThermodynamicState{FT}) where {FT<:Real} =
```

Comment:
Instead of retyping the linearized pressure for dry air in l. 110, wouldn't it be better for dry air to call the general form with all specific humidities zero? (That way, we have to change only one line if we need to change this.)

Reply:
True. It's a trade-off between fewer flops with the simplified/short-circuited expressions for the dry case vs more flops but re-using the general expressions.

Reply:
For now, it's likely fine to re-use, and when we go to optimize we can make these simplifications.